notebooks/fairness-dashboard-loan-allocation.ipynb

```diff
@@ -11,39 +10,40 @@
     "For the purposes of this notebook, we shall treat this as a loan decision problem. We will pretend that the label indicates whether or not each individual repaid a loan in the past. We will use the data to train a predictor to predict whether previously unseen individuals will repay a loan or not. The assumption is that the model predictions are used to decide whether an individual should be offered a loan.\n",
     "\n",
     "We will first train a fairness-unaware predictor and use the fairness dashboard to demonstrate how this model leads to unfair decisions (under a specific notion of fairness called *demographic parity*). We then mitigate unfairness by applying the `GridSearch` algorithm from `Fairlearn` package."
-   ]
+   ],
+   "metadata": {}
```

Comment:
It doesn't look like there are significant changes to this notebook? If so, can it just be removed from the PR?